zuul.d/jobs.yaml

```diff
@@ -2072,16 +1977,6 @@
     vars:
       ansible_test_network_cli_ssh_type: libssh
 
-- job:
```

Comment:
Shouldn't we keep testing local on stable branches?  I think for vyos, we should be safe